src/parameters.rs

```diff
@@ -55,42 +40,32 @@ where
     directory.push("proving");
     fs::create_dir_all(&directory).ok()?;
     let mint = directory.join("mint.dat");
```

Comment:
I think we should also rename data file, right?

Reply:
im not sure in this case